cmd/system-probe/api/module/factory_linux.go

```diff
@@ -17,4 +17,8 @@ type Factory struct {
 	ConfigNamespaces []string
 	Fn               func(cfg *sysconfigtypes.Config, deps FactoryDependencies) (Module, error)
 	NeedsEBPF        func() bool
+
+	// IgnoreForSuccessCheck can be set to true if system-probe should not consider this module when checking
+	// if at least one module was successfully loaded.
+	IgnoreForSuccessCheck bool
```

Comment:
1. I think we should rename this variable
2. I don't think you should tie together the new module and this change in a single PR

Reply:
This change is very tightly coupled with the rest of the PR, and as it was small I preferred to put it here. Regarding the name, I agree it's not the best but I couldn't thing of anything better.